src/main/app/src/Components/Landscape/Map/Map.tsx

```diff
@@ -270,6 +267,16 @@ const Map: React.FC<Props> = () => {
             </ReactSVGPanZoom>
           )}
         />
+        <Toolbar className={classes.bottomBar}>
+          {landscape && assessments && (
+            <Dashboard
+              findItem={findItem}
+              setSidebarContent={setSidebarContent}
+              landscape={landscape}
+              assessments={assessments}
+            />
+          )}
```

Comment:
I thought the dashboard had been removed. What is this then? Is this the container for the buttons on the bottom of a map?

Reply:
I would rename this to something like e.g. StatusLegend or something that better represents what this actually is (also `DashboardLayout`. In addition, 

I just saw that this is just the Toolbar I was talking about in one of the other comments (I wouldn't allow the map to be visible below it).